microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -183,22 +176,58 @@ This article gives you a lot to learn. But now you have a solid foundation in th
 
 Using the workflows and tools that you just learned, click **Optimize** on the demo to turn on the optimized code, take another performance recording, and then analyze the results.  From the improved framerate to the reduction in events in the flame chart in the **Main** section, the optimized version of the app does much less work, resulting in better performance.
 
-The optimized code uses a different sequence of actions to do less work. Note that this code could be made even faster by only using properties that only affect compositing, instead of manipulating the `top` property of every icon.
-<!--  > For more information, see [Use transform and opacity changes for animations](https://web.dev/stick-to-compositor-only-properties-and-manage-layer-count/#use-transform-and-opacity-changes-for-animations). todo: add rendering section when available -->
-
+Compare this snippet of JavaScript from the unoptimized version of the app:
+
+```javascript
+var pos = m.classList.contains("down")
+  ? m.offsetTop + distance
+  : m.offsetTop - distance;
+if (pos < 0) pos = 0;
+if (pos > maxHeight) pos = maxHeight;
+m.style.top = pos + "px";
+if (m.offsetTop === 0) {
+  m.classList.remove("up");
+  m.classList.add("down");
+}
+if (m.offsetTop === maxHeight) {
+  m.classList.remove("down");
+  m.classList.add("up");
+}
+```
+
+`m` references each blue icon on the page. In the unoptimized version, we refer to `pos` as the new vertical position of the icon, calculated based on `m.offsetTop + distance` if the icon is moving down and where `distance` is a fixed increment.  If the icon is moving up instead, the new position is instead calculated with `m.offsetTop - distance`.
+
+We check if we've exceeded either `0` or `maxHeight` as the new position whether we're moving up or down, adjust accordingly, and then we finally set the new position for each icon in CSS with `m.style.top = pos + "px";`.
+
+The optimized code uses a different sequence of actions to do less work. Here is the same snippet of JavaScript from the optimized version of the app: 
```

Comment:
global, as many as 7 spots: change "unoptimized" to "non-optimized"?  I lean towards non- but not sure how to justify via style guides.  Does UI say "unoptimized" on a button label?

Reply:
<img width="235" alt="image" src="https://user-images.githubusercontent.com/34316555/222297087-aafc243e-0c85-44d1-8556-95d32e17e569.png">

When you click the "Optimize" button in the demo site, the button changes to say "Un-optimize". I can match the button or we can change what the button says since @captainbrosset was proposing updating the demo code to make it more readable.

Reply:
There seems to be way more web search hits on 'unoptimized' than "non-optimized".  I'd encourage going with whatever phrasing actual users in the field/industry would actually write & say & recognize.